test/e2e/common_test.go

```diff
@@ -64,6 +64,9 @@ func defaultShoot(generateName string) *gardencorev1beta1.Shoot {
 					RegistryBurst:       pointer.Int32(20),
 				},
 				KubeAPIServer: &gardencorev1beta1.KubeAPIServerConfig{},
+				VerticalPodAutoscaler: &gardencorev1beta1.VerticalPodAutoscaler{
+					Enabled: true,
+				},
```

Comment:
No need to enable it explicitly, there is admission plugin in gardener that should do it when the field is nil. Ref https://github.com/gardener/gardener/blob/00753693013505a2ea9f9350c8d385f2c2c166e6/plugin/pkg/shoot/vpa/admission.go#L63-L65